src/test/java/uk/gov/pay/adminusers/resources/UserResourcePatchIT.java

```diff
@@ -136,4 +142,102 @@ void shouldError_whenPatchRequiredFieldsAreMissing() {
+    }
+
+
+    @Test
+    void shouldRemoveFeatureFromUser_whenOpIsRemove () {
+        String username = randomUuid();
+        String email = username + "@example.com";
+        User userWithFeatures = userDbFixture(databaseHelper).withEmail(email).withFeatures("test_feature_1,test_feature_2").insertUser();
+        
+        JsonNode payload = mapper.valueToTree(Map.of("path", "features", "op", "remove", "value", "test_feature_1"));
+
+        String featureList = givenSetup()
+                .when()
+                .contentType(JSON)
+                .accept(JSON)
+                .body(payload)
+                .patch(format(USER_RESOURCE_URL, userWithFeatures.getExternalId()))
+                .then()
+                .extract().path("features");
+
+        Set<String> featureSet = new HashSet<>(List.of(featureList.split(",")));
+        featureSet.remove("");
+        assertThat(featureSet.size(), is(1));
+        assertThat(featureSet, contains("test_feature_2"));
+    }
+
+    @Test
+    void shouldRemoveMultipleFeaturesFromUser_whenListOfFeaturesIsProvided () {
+        String username = randomUuid();
+        String email = username + "@example.com";
+        User userWithFeatures = userDbFixture(databaseHelper).withEmail(email).withFeatures("test_feature_1,test_feature_2,test_feature_3,test_feature_4").insertUser();
+
+        JsonNode payload = mapper.valueToTree(Map.of("path", "features", "op", "remove", "value", "test_feature_1,test_feature_4"));
+
+        String featureList = givenSetup()
+                .when()
+                .contentType(JSON)
+                .accept(JSON)
+                .body(payload)
+                .patch(format(USER_RESOURCE_URL, userWithFeatures.getExternalId()))
+                .then()
+                .extract().path("features");
+        
+        Set<String> featureSet = new HashSet<>(List.of(featureList.split(",")));
+        featureSet.remove("");
+        assertThat(featureSet.size(), is(2));
+        assertThat(featureSet, containsInAnyOrder("test_feature_2", "test_feature_3"));
+    }
+
+    @Test
+    void shouldResultInEmptyFeaturesList_whenAllFeaturesAreRemoved () {
+        String username = randomUuid();
+        String email = username + "@example.com";
+        User userWithFeatures = userDbFixture(databaseHelper).withEmail(email).withFeatures("test_feature_1,test_feature_2,test_feature_3,test_feature_4").insertUser();
+
+        JsonNode payload = mapper.valueToTree(Map.of("path", "features", "op", "remove", "value", "test_feature_1,test_feature_2,test_feature_3,test_feature_4"));
+
+        givenSetup()
+                .when()
+                .contentType(JSON)
+                .accept(JSON)
+                .body(payload)
+                .patch(format(USER_RESOURCE_URL, userWithFeatures.getExternalId()))
+                .then()
+                .body("features", is(""));
+    }
```

Comment:
Probably want a test for when a feature that doesn't exist is removed. I imagine this will do nothing to the existing set of features. It should be logged though.